io/common/virtual_bind_unbind.py.data/README.txt

```diff
@@ -0,0 +1,12 @@
+This script will unbind and bind the virtualized device
+for vnic, vscsi, vfc,veth as many times as specified by 
+the user
+
+Inputs Needed :
+------------------------------------
+virtual_device - virtual device for the test interface or disk name.
+virtual_slot - slot name for the virtual device, which can get it from lscfg -vl command.
```

Comment:
update the README to get slot for vfc use lsslot, as vfc do not give correct slot number using lscfg -vl ( all other vdevices gives correct slot using lscfg)

```
[root@ltc-zz1b-lp1 ~]# lscfg -vl sdm
  sg4 1:0:1:0 sdm  **H1-B0-T1-L0**           16 BitVirtual I/O Device - SCSI
                                         Disk Drive
        Manufacturer Name.........IBM
        Machine Type-Model........2145
        Alterable ROM Level.......0000
        SCSI Host.(XH)............1
        SCSI Bus.(XB).............0
        SCSI Target.(XT)..........1
        SCSI Lun.(XL).............0
        First 8 bytes of SCSI query response.(Z0)
        ..........................00000007008083B0
        Device Specific.(Z7)......60050768028211D8
        Location Code.(YL)........H1-B0-T1-L0

[root@ltc-zz1b-lp1 ~]# lsslot
# Slot                     Description       Linux Name    Device(s)
U9009.42A.13C6F1W-V2-C0    Virtual I/O Slot  30000000      vty
U9009.42A.13C6F1W-V2-C2    Virtual I/O Slot  30000002      l-lan
U9009.42A.13C6F1W-V2-C201  Virtual I/O Slot  300000c9      vfc-client
```